msal/src/main/java/com/microsoft/identity/client/internal/controllers/OperationParametersAdapter.java

```diff
@@ -180,14 +180,14 @@ public static AcquireTokenOperationParameters createAcquireTokenOperationParamet
             acquireTokenOperationParameters.setAuthorizationAgent(AuthorizationAgent.DEFAULT);
         }
 
-        if (acquireTokenParameters.getUiBehavior() == null) {
+        if (acquireTokenParameters.getPrompt() == null) {
             acquireTokenOperationParameters.setOpenIdConnectPromptParameter(
                     OpenIdConnectPromptParameter.SELECT_ACCOUNT
             );
         } else {
             acquireTokenOperationParameters.setOpenIdConnectPromptParameter(
                     acquireTokenParameters
-                            .getUiBehavior()
+                            .getPrompt()
```

Comment:
Ok, so when Prompt is WhenRequired , may be we should not set it to operation parameters here right? As this conversion here would throw an exception ?

Reply:
Good catch.  I'll fix.  Thanks